### ISSUE TITLE
Checking all scores for music21.converter.parse(filePath, format='romantext'). EDIT: Wrong flat symbol (`-`) in many BPS-FH scores 

### ISSUE THREAD

**Issue by napulen** [MarkGotham/When-in-Rome#4]
Performing a basic check for music21-parsing (`music21.converter.parse(filePath, format='romantext')`) on all scores described in this repository. Most scores are parsed without errors, however, the following list throw exceptions:

```python
[
    'BPS-FH_27-i.txt',
    'BPS-FH_10-i.txt',
    'BPS-FH_01-i.txt',
    'BPS-FH_12-i.txt', 
    'BPS-FH_13-i.txt', 
    'BPS-FH_29-i.txt', 
    'BPS-FH_17-i.txt', 
    'BPS-FH_06-i.txt', 
    'BPS-FH_03-i.txt', 
    'BPS-FH_04-i.txt', 
    'BPS-FH_08-i.txt', 
    'BPS-FH_02-i.txt',
    'BPS-FH_23-i.txt', 
    'BPS-FH_16-i.txt', 
    'BPS-FH_22-i.txt', 
    'BPS-FH_25-i.txt', 
    'BPS-FH_11-i.txt', 
    'BPS-FH_21-i.txt', 
    'BPS-FH_26-i.txt', 
    'BPS-FH_30-i.txt', 
    'BPS-FH_18-i.txt', 
    'BPS-FH_07-i.txt', 
    'BPS-FH_32-i.txt', 
    'BPS-FH_31-i.txt', 
    'BPS-FH_05-i.txt', 
    'BPS-FH_19-i.txt'
]
```
Haven't looked into the root cause yet.

More to come on this.

**Comment by napulen** [comment]
For `BPS-FH_27-i.txt`, it seems to be related with flats spelled with a hyphen `-` rather than `b`. 

Possibly the same issue in all of these scores.

Just to double check, RomanText does not support flats (`-`) in key or chord spellings? I thought it did.

**Comment by napulen** [comment]
After correcting all the keys with `-` flat symbols, still getting a (shorter) list of scores that won't parse:
```python
['BPS-FH_01-i.txt', 
'BPS-FH_12-i.txt', 
'BPS-FH_17-i.txt', 
'BPS-FH_03-i.txt', 
'BPS-FH_02-i.txt', 
'BPS-FH_23-i.txt', 
'BPS-FH_30-i.txt', 
'BPS-FH_07-i.txt', 
'BPS-FH_19-i.txt']
```

I haven't figured out the root cause of these.

**Comment by napulen** [comment]
I have at least verified that `BPS-FH_01-i.txt` is a different type of issue: wrong beat information.

Each of the remaining files in this list is probably going to require its own proofreading.

I am going to open a separate issue to pinpoint the specifics of these conflicting files. 

The scope of this issue will then be about the `-` flat symbols. A PR for that has been already sent.

**Comment by napulen** [comment]
`-` vs `b` issues fixed with #5
